spec/index.html

```diff
@@ -2383,12 +2381,127 @@ <h4>YamlLdErrorCode</h4>
           </section>
         </section>
 
+        <section>
+          <h2>Implementations</h2>
+
+          TODO
+        </section>
+
       </section>
 
       <section>
         <h2>Convert Extended YAML-LD to Basic YAML-LD and back</h2>
 
-        TODO
+        <p>
+          This approach is simpler than the Extended Internal Representation because it
+          does not require any changes to the internal structures of existing JSON-LD
+          libraries.
+        </p>
+
+        <p>
+          Instead, we implement two API functions:
+        </p>
+
+        <dl data-sort>
+          <dt><code>extended_to_basic(extended_document: YAML-LD) → YAML-LD</code></dt>
+          <dd>
+            Converts an Extended YAML-LD document to a JSON-LD document that a standard
+            processor can work with
+          </dd>
+
+          <dt><code>basic_to_extended(basic_document: JSON-LD | YAML-LD) → YAML-LD</code></dt>
+          <dd>
+            Converts a basic JSON-LD or YAML-LD document to the Extended form, making
```

Comment:
This needs some discussion of how JSON-LD gets involved in "converting Extended YAML-LD to Basic YAML-LD and back".

Reply:
It shouldn't need to go through JSON.